iohk-monitoring/src/Cardano/BM/Backend/Log.lhs

```diff
@@ -335,7 +335,7 @@ passN backend katip (LogObject loname lometa loitem) = do
                                 , _itemEnv       = env ^. KC.logEnvEnv
                                 , _itemSeverity  = sev2klog sev
                                 , _itemThread    = threadIdText
-                                , _itemHost      = env ^. KC.logEnvHost
+                                , _itemHost      = unpack $ hostname lometa -- env ^. KC.logEnvHost
```

Comment:
Do we need this comment `-- env ^. KC.logEnvHost`?

Reply:
thanks for spotting this. will remove it.